src/interfaces/routes.ts

```diff
@@ -28,8 +28,13 @@ export interface GetTransferHistoryRequest {
   action?: string; //Deposit / Withdraw
 }
 export interface GetOrderRequest extends GetTransactionHistoryRequest {
+  symbol?: MarketSymbol;
+  orderId?: number;
+  orderHashes?: string[];
```

Comment:
Should `orderId` also be a list?

Reply:
yes but we're eventually going to remove this `orderId` param as we don't use it anywhere. That's why I didn't update it on dapi, as I have to inform everyone just in case anyone's using it.